sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/_client.py

```diff
@@ -15,7 +15,7 @@
 from azure.core.rest import HttpRequest, HttpResponse
 
 from ._configuration import RadiologyInsightsClientConfiguration
-from ._operations import RadiologyInsightsClientOperationsMixin
+from ._operations._patch import RadiologyInsightsClientOperationsMixin
```

Comment:
We shouldn't make manual updates in the generated files. We should use the patch files instead. I think we'll have to patch the client to properly expose the updated method signatures.